docs/plugins/testing_workshop_docs/Pytest-testing-frameworks.md

```diff
@@ -0,0 +1,104 @@
+# Pytest testing framework
+
+This article will be more valuable if you are familiar with the Python programming language and Napari software. It is the second in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 9:21. This article is a summary of the information in the video and should stand on its own. The other articles are:  
+* Article 1: [Python’s assert keyword](./Pythons-assert-keyword.md) 
+* Article 2: This article  
+* Article 3: [Readers and fixtures](./Readers-and-fixtures)  
+* Article 4: [Test coverage](./Test-coverage)  
+* Article 5: Testing widgets  
+  
+#### This article covers:  
+* [Testing framework features](#testing-framework-features)  
+* [Parametrization](#parametrization)  
+
+#### Resources  
+The example plugin and all the tests discussed in this article are available in [this GitHub repository](https://github.com/DragaDoncila/plugin-tests).
```

Comment:
Because of how these documents are structures, it is highly recommended to use only continuous levels of headers. For example, if the title is `# Title` the next header should be `## Header` and not `#### Header` as it i here. My formatting suggestion would be to use bold instead, especially if this doesn't need to show up in the right-hand side toc:

```suggestion
**This article covers:**
* [Testing framework features](#testing-framework-features)  
* [Parametrization](#parametrization)  

**Resources**
The example plugin and all the tests discussed in this article are available in [this GitHub repository](https://github.com/DragaDoncila/plugin-tests).
```

Reply:
I will look at this.  I think it needs to stay in the toc.  I just thought it looked too big as ###.  Thanks.

Reply:
Added a ## heading for Article information to all articles.  changed these headings to ###.